### PR TITLE
Fix Simple Injector

### PR DESCRIPTION
**Pull request by Gelphy**
Some changes to packages names, one Simple Injector package for one module, SimpleInjectorConfiguration moved to EntryPoint,  changes with owin settings
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/74%23discussion_r60042992%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/74%23discussion_r60044010%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/74%23discussion_r60046115%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/74%23discussion_r60046533%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/74%23discussion_r60046618%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/74%23discussion_r60049206%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/74%23discussion_r60051059%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%20cc4429e00544c39b4b1fb83f2a785447e9438083%20DevelopGitHub/GitHubExtension/GithubExtension.Extensions/UseOwinContextInjectorExtension.cs%2012%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/74%23discussion_r60042992%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Can%20you%20explain%20why%20you%20need%20this%3F%22%2C%20%22created_at%22%3A%20%222016-04-18T11%3A33%3A11Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6068068%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/TerraVenil%22%7D%7D%2C%20%7B%22body%22%3A%20%22Use%20this%20extension%20method%20for%20correct%20work%20owin%20with%20simple%20injector%2C%20according%20to%20this%20tutorial%20http%3A//simpleinjector.readthedocs.org/en/latest/owinintegration.html%22%2C%20%22created_at%22%3A%20%222016-04-18T11%3A44%3A52Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/10926945%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Gelphy%22%7D%7D%2C%20%7B%22body%22%3A%20%22What%20was%20the%20issue%20in%20the%20project%20which%20solved%20this%20configuration%3F%22%2C%20%22created_at%22%3A%20%222016-04-18T12%3A04%3A18Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6068068%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/TerraVenil%22%7D%7D%2C%20%7B%22body%22%3A%20%22There%20was%20some%20troubles%20with%20creatation%20ApplicationUserManager%20per%20owin%20context%22%2C%20%22created_at%22%3A%20%222016-04-18T12%3A08%3A19Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/10926945%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Gelphy%22%7D%7D%2C%20%7B%22body%22%3A%20%22Can%20you%20show%20me%20error%20message%3F%22%2C%20%22created_at%22%3A%20%222016-04-18T12%3A09%3A12Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6068068%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/TerraVenil%22%7D%7D%2C%20%7B%22body%22%3A%20%22is%20registered%20as%20%27Web%20API%20Request%27%20lifestyle%2C%20but%20the%20instance%20is%20requested%20outside%20the%20context%20of%20a%20Web%20API%20Request.%22%2C%20%22created_at%22%3A%20%222016-04-18T12%3A31%3A59Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/10926945%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Gelphy%22%7D%7D%2C%20%7B%22body%22%3A%20%22Ok.%22%2C%20%22created_at%22%3A%20%222016-04-18T12%3A47%3A20Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6068068%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/TerraVenil%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20DevelopGitHub/GitHubExtension/GithubExtension.Extensions/UseOwinContextInjectorExtension.cs%3AL1-22%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull cc4429e00544c39b4b1fb83f2a785447e9438083 DevelopGitHub/GitHubExtension/GithubExtension.Extensions/UseOwinContextInjectorExtension.cs 12'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-013/pull/74#discussion_r60042992'>File: DevelopGitHub/GitHubExtension/GithubExtension.Extensions/UseOwinContextInjectorExtension.cs:L1-22</a></b>
- <a href='https://github.com/TerraVenil'></a> Can you explain why you need this?
- <a href='https://github.com/Gelphy'></a> Use this extension method for correct work owin with simple injector, according to this tutorial http://simpleinjector.readthedocs.org/en/latest/owinintegration.html
- <a href='https://github.com/TerraVenil'></a> What was the issue in the project which solved this configuration?
- <a href='https://github.com/Gelphy'></a> There was some troubles with creatation ApplicationUserManager per owin context
- <a href='https://github.com/TerraVenil'></a> Can you show me error message?
- <a href='https://github.com/Gelphy'></a> is registered as 'Web API Request' lifestyle, but the instance is requested outside the context of a Web API Request.
- <a href='https://github.com/TerraVenil'></a> Ok.

<a href='https://www.codereviewhub.com/ITsvetkoFF/Kv-013/pull/74?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/ITsvetkoFF/Kv-013/pull/74?approve=1'></a>&nbsp;<a href='https://github.com/ITsvetkoFF/Kv-013/pull/74'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by ITsvetkoFF** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/ITsvetkoFF/Kv-013/pull/74'>Refresh</a>.<a href='#crh-update-none'></a>

File: DevelopGitHub/GitHubExtension/GithubExtension.Extensions/UseOwinContextInjectorExtension.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+using SimpleInjector.Extensions.ExecutionContextScoping;
+
+namespace GithubExtension.Extensions
+{
+    public static class UseOwinContextInjectorExtension
+    {
+        public static void UseSimpleInjectorContext(this IAppBuilder app, Container container)
+        {
+            // Create an OWIN middleware to create an execution context scope
+            app.Use(async (context, next) =>
```
Comment (TerraVenil) [inline DevelopGitHub/GitHubExtension/GithubExtension.Extensions/UseOwinContextInjectorExtension.cs:12]: Can you explain why you need this?

Comment (Gelphy) [inline DevelopGitHub/GitHubExtension/GithubExtension.Extensions/UseOwinContextInjectorExtension.cs:12]: Use this extension method for correct work owin with simple injector, according to this tutorial http://simpleinjector.readthedocs.org/en/latest/owinintegration.html

Comment (TerraVenil) [inline DevelopGitHub/GitHubExtension/GithubExtension.Extensions/UseOwinContextInjectorExtension.cs:12]: What was the issue in the project which solved this configuration?

Comment (Gelphy) [inline DevelopGitHub/GitHubExtension/GithubExtension.Extensions/UseOwinContextInjectorExtension.cs:12]: There was some troubles with creatation ApplicationUserManager per owin context

Comment (TerraVenil) [inline DevelopGitHub/GitHubExtension/GithubExtension.Extensions/UseOwinContextInjectorExtension.cs:12]: Can you show me error message?

Comment (Gelphy) [inline DevelopGitHub/GitHubExtension/GithubExtension.Extensions/UseOwinContextInjectorExtension.cs:12]: is registered as 'Web API Request' lifestyle, but the instance is requested outside the context of a Web API Request.

Comment (TerraVenil) [inline DevelopGitHub/GitHubExtension/GithubExtension.Extensions/UseOwinContextInjectorExtension.cs:12]: Ok.

**Pull request merged**